models/users.models.js

```diff
@@ -0,0 +1,10 @@
+const db = require('../db/connection');
+
+function selectUsers(){
+return db.query('SELECT username,name,avatar_url FROM users;')
```

Comment:
I would recommend grabbing the user_id aswell because you might need it if you decide to allow users to edit their profiles in the frontend